src/util/browser.js

```diff
@@ -3,31 +3,39 @@
  * https://github.com/videojs/video.js/blob/4238f5c1d88890547153e7e1de7bd0d1d8e0b236/src/js/utils/browser.js
  */
 
-// User agent example taken from Safari desktop:
-const useAgent = navigator && navigator.userAgent || '';
+/**
+* Retrieve from the navigator the user agent property.
+* @returns user agent property.
+*/
+function getNavigator(){
```

Comment:
@echatzief Can you please change this to getUserAgent() ?

Reply:
Sure. I changed it